### PR TITLE
Add Installation guide for installing the collector using Ansible

### PR DESCRIPTION
**Pull request by ishanjainn**
This PR adds instructions for installing OpenTelemetry Collector using [`grafana.grafana`](https://github.com/grafana/grafana-ansible-collection) Ansible role. This role is prepackaged with the Ansible Binary so doesn't need any installation which means users can directly use this playbook to scale and manage their Collectors across multiple Instance.

### CODE REVIEW

**Comment by svrnm** [comment]
> Hey @svrnm Yeah I had a similar thought around this and my initial preference was also `Option 1` but that might take a while to get implemented as we would have to create a fresh new namespace in ansible galaxy and the review would also take time.

Agreed, but the earlier you start with it, the sooner we are there:-) So I can only advice to get in touch with the collector SIG to discuss this!
 
> Hence went for `Option 2` with the goal being that this role for the collector would be quickly available for people + the `grafana.grafana` collection is already registered in ansible galaxy and the collection does not need to be separately installed by a user (it comes prebuilt when anyone install ansible) but the goal being to have this in docs though as thats where users would find this more!
> 
> and yup, Open to doing a blogpost about it and registry too for now!

Those 2 options are not mutually exclusive, so we can start with that, and then grow from there. Adding to the registry is the quickest thing, since it's just a file you need to update. Happy to review the blog post, note that we have an influx of many posts right now due to KubeCon, so the earliest we can have that is probably post KubeCon

File: content/en/docs/collector/installation.md [hunk 1]
```diff
@@ -97,6 +97,58 @@ For guidance on how to use the Collector with Kubernetes, see
 You can find reference job files to deploy the Collector as an agent, gateway,
 and as full demo in [Getting Started with OpenTelemetry on HashiCorp Nomad][].
 
+## Ansible
+
+Deploy the OpenTelemetry Collector on your instances using Ansible with the following playbook. This playbook utilizes the [Grafana Ansible role](https://galaxy.ansible.com/ui/repo/published/grafana/grafana/content/role/opentelemetry_collector/), aiding in provisioning and managing multiple OpenTelemetry Collectors.
```
Comment (theletterf) [inline content/en/docs/collector/installation.md:102]:
```suggestion
Deploy the OpenTelemetry Collector on your instances using the
following Ansible playbook. This playbook uses the
[Grafana Ansible role](https://galaxy.ansible.com/ui/repo/published/grafana/grafana/content/role/opentelemetry_collector/),
aiding in provisioning and managing multiple OpenTelemetry Collectors.
```

File: content/en/docs/collector/installation.md [hunk 2]
```diff
@@ -97,6 +97,58 @@ For guidance on how to use the Collector with Kubernetes, see
...
+              receivers: [otlp]
+              processors: [batch]
+              exporters: [otlp]
+            logs:
+              receivers: [otlp]
+              processors: [batch]
+              exporters: [otlp]
+```
+
+This playbook sets up the OpenTelemetry Collector with basic configurations for receiving, processing, and exporting telemetry data. Customize the configuration as needed to fit your environment and requirements.
```
Comment (theletterf) [inline content/en/docs/collector/installation.md:150]:
```suggestion
The previous Ansible playbook sets up the OpenTelemetry Collector with a basic set of receivers, processors, and exporters. Customize the configuration as needed to fit your environment and requirements.
```

**Review by theletterf — CHANGES_REQUESTED** [review]

**Comment by svrnm** [comment]
>     1. The blog post, Where should I add the blogpost , [Here](https://github.com/open-telemetry/opentelemetry.io/tree/main/content/en/blog/2024) ?

Yes, you can use `npx hugo new content/en/blog/2024/<name>.md` to get a file auto generated for you. Note that the blog post should be self-contained, so you need to add some words around why ansible, etc, maybe even give people a end-to-end tutorial how they can get started with it

> #4139 -> here is the PR for the registry

reviewed

**Comment by mx-psi** [comment]
I am happy to help with a potential donation, so long as we have adequate staffing from the current owners to kickstart the maintainers/approvers group of this Ansible playbook :)

**Comment by mx-psi** [comment]
@ishanjainn If you can come to the Collector SIG tomorrow (see links [here](https://github.com/open-telemetry/community#implementation-sigs)) and explain this I think that would be the best. You can also create an issue in the community repo before the meeting so we have a place to comment on :)

**Comment by svrnm** [comment]
> Hey @mx-psi For the donation process, I am thinking of maintaining the role at both places (the grafana repo can carry the symlink basically). Should I just create the issue in the community repo or anything else needs to be done?

what @mx-psi said, attend the collector meeting and go ahead to create the community issue. Please take a close look at the requirements for a donation, if anything appears problematic to you, let us know.

A thing I can send upfront is the following: Ansible Galaxy requires GPL3-compatible licensing of your code, which Apache-2.0 is not, so _if_ we are going to accept this code, it needs to be GPL3 as well (as it is right now) or it needs to be relicensed to a compatible license (BSD, etc, afaik)

**Comment by svrnm** [comment]
> Cool @svrnm @mx-psi, I might not make it tomorrow, Will get into the next week's one but ill create the issue tommorow and all that stuff started. and yeah license wise we should be good as is (GPL-3.0)

We are good with Ansible Galaxy, but not with our current donation requirements:

> All donated code requires a license compatible with the Apache Software License 2.0, and donated code will require a change of copyright to reflect the OpenTelemetry Authors. 

But this is something we can discuss during the process. I just wanted to highlight this, so it does not come as a surprise that we need to figure that out.

**Comment by svrnm** [comment]
I close this PR as it is going to be replaced by the blog post. If this is incorrect please reopen it

**Pull request closed without merging**